app/fields/person_photo_field.rb

```diff
@@ -0,0 +1,7 @@
+require "administrate/field/base"
+
+class PersonPhotoField < Administrate::Field::Base
+  def to_s
+	    data
+  end
```

Comment:
Same comment as the previous `to_s`.

Reply:
Has no visible effect, so am changing as requested.